### DISCUSSION TITLE
Displacement list

### DISCUSSION THREAD

**Discussion by janetunde** [JWock82/Pynite#138] [Q&A] [answered]
Apologies, one follow-up question. How would I view the lists of displacements, torques, moments etc along each submember unit of the member? Rather than just plotting them? I am having trouble using the torque, shear etc functions to access information about each submember created by the descritize function, as described here: https://raw.githubusercontent.com/JWock82/PyNite/master/PyNite/PhysMember.py.
Thank you again.

**Comment by JWock82** [comment, marked as answer]
By default, members in PyNite are treated as "Physical Members". A physical member will automatically descretize itself into sub-members at any nodes that occur along the length of the member. It saves the user from having to think about descritizing the member, and it models how beams and columns are actually built in the real world.

Here's an example of how to get the deflection for member M3 at a location of 10 units from the start of the physical member for the load combination '1.2D+1.6L' from a model named `model`:

`model.Members['M3'].deflection('dy', x=10, combo_name='1.2D+1.6L')`

It sounds like you may be interested in results for the sub-members themselves, rather than the overall combined physical member. If that's the case, just descretize the model yourself by making a series of physical members, rather than one long physical member.

**Reply by janetunde** [reply]
Thank you!
